src/main.ts

```diff
@@ -103,12 +125,42 @@ function watch(config: webpack.Config, options: WebpackOptions, args: BuildArgs)
 	})(config.entry);
 
 	const compiler = webpack(config);
+	const portRange = String(args.port || '9999:9990');
+	let ports: number[] = [];
+	let serverPort: number | undefined;
+
+	if (portRange.indexOf(':') >= 0) {
+		let [low, high] = portRange.split(':').map(p => parseInt(p, 10));
+
+		if (high < low) {
+			[low, high] = [high, low];
```

Comment:
We don't cuddle in arrays or objects i.e. `[low, high]` or `{foo: 'bar'}` should be `[ low, high ]` or `{ foo: 'bar' }`